tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-torch1100-cuda102.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
+
+stages:
+- template: templates/py-packaging-training-cuda-stage.yml
+  parameters:
+    build_py_parameters: --enable_training --update --build
+    torch_version: '1.10.0'
+    cuda_version: '10.2'
+    gcc_version: 10
+    cmake_cuda_architectures: 37;50;52;60;61;70;75;80;86
+    docker_file: Dockerfile.manylinux2014_training_cuda11_3
```

Comment:
The file name is cuda102 but the actual cuda version is 11_3?

Reply:
Updated to use cuda 10_2 dockerfile.